### DISCUSSION TITLE
How do you set the default font for the notes?

### DISCUSSION THREAD

**Discussion by thupten1234** [freeplane/freeplane#914] [Help]
Is there a way to set the default font for the notes? I don't mean node font.

**Comment by sgjohnson1981** [comment]
The only thing I could find was a setting in Tools -> Preferences -> Appearance -> Look and Feel that sets the default note font to the default general font. So doesn't seem like you can set it individually.

**Comment by sgjohnson1981** [comment]
It's not as fast as using keys to create nodes, but an alternative would be to use a template node and use CTRL + drag and release it wherever you want it. That copies it rather than moving it.

**Comment by abc16361** [comment]
For the current working map, go to Format -> Manage Styles -> Edit styles, there you can do the note font setting (please see the screenshot). For all maps, do the same on the templates which are stored in the templates folder (C:\Program Files\Freeplane\resources\templates). Good luck!
![noteFontSetting](https://user-images.githubusercontent.com/101362122/210021487-9967e075-1a02-4d8a-b004-4c9dbcd61426.png)
